compiler/daml-lf-tools/tests/DA/Daml/LF/Simplifier/Tests.hs

```diff
@@ -22,9 +22,7 @@ main = defaultMain $ testGroup "DA.Daml.LF.Simplifier"
     -- The Simplifier calls the typechecker whose behavior is affected by feature flags. The
     -- simplifier may thus behave differently based on the version of LF and thus we need to test
     -- both x.dev versions as they will diverge over time.
```

Comment:
I'd move this comment to `constantLiftingTests`, removing the part about "both x.dev versions"

Reply:
Good catch! Done.